exercises/practice/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -12,32 +12,15 @@ import (
ense. But how do you know, not only that it is possible, but also that it is desirable to reform man in that way? And what leads you to the conclusion that man's inclinations need reforming? In short, how do you know that such a reformation will be a benefit to man? And to go to the root of the matter, why are you so positively convinced that not to act against his real normal interests guaranteed by the conclusions of reason and arithmetic is certainly always advantageous for man and must always be a law for mankind? So far, you know, this is only your supposition. It may be the law of logic, but not the law of humanity. You think, gentlemen, perhaps that I am mad? Allow me to defend myself. I agree that man is pre-eminently a creative animal, predestined to strive consciously for an object and to engage in engineering - that is, incessantly and eternally to make new roads, wherever they may lead. But the reason why he wants sometimes to go off at a tangent may just be that he is predestined to make the road, and perhaps, too, that however stupid the "direct" practical man may be, the thought sometimes will occur to him that the road almost always does lead somewhere, and that the destination it leads to is less important than the process of making it, and that the chief thing is to save the well-conducted child from despising engineering, and so giving way to the fatal idleness, which, as we all know, is the mother of all the vices. Man likes to make roads and to create, that is a fact beyond dispute. But why has he such a passionate love for destruction and chaos also? Tell me that! But on that point I want to say a couple of words myself. May it not be that he loves chaos and destruction (there can be no disputing that he does sometimes love it) because he is instinctively afraid of attaining his object and completing the edifice he is constructing? Who knows, perhaps he only loves that edifice from a distance, and is by no means in love with it at close quarters; perhaps he only loves building it and does not want to live in it, but will leave it, when completed, for the use of les animaux domestiques - such as the ants, the sheep, and so on. Now the ants have quite a different taste. They have a marvellous edifice of that pattern which endures for ever - the ant-heap. With the ant-heap the respectable race of ants began and with the ant-heap they will probably end, which does the greatest credit to their perseverance and good sense. But man is a frivolous and incongruous creature, and perhaps, like a chess player, loves the process of the game, not the end of it. And who knows (there is no saying with certainty), perhaps the only goal on earth to which mankind is striving lies in this incessant process of attaining, in other words, in life itself, and not in the thing to be attained, which must always be expressed as a formula, as positive as twice two makes four, and such positiveness is not life, gentlemen, but is the beginning of death.`
```

Comment:
@andrerfcsantos What do you think about the formatting here? Should line breaks be used? Not sure how the editor will display this on the website.

(I will run the benchmarks again later.)

Reply:
I was wondering the same thing. I think we should either put all the texts in one line to allow an easier access to the test code, or split it into newlines with 80ish characters per line to make the strings more readable. I'm fine with either option more tempted to go with the second one, as the texts we have are not **that** big.

Reply:
reduced the max character count per line to around 100 charcters